### DISCUSSION TITLE
[A/L] (2021/22) Relazioni - Esercizio 20

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#292] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152320365-7be99cae-4de1-496c-a617-4c24d61acc6c.png)

**Comment by LuginiAndrea** [comment]
1) $R^{\cap} = \left(1,2\right), \left(2,3\right), \left(1,3\right)$. Vediamo subito la transitività
2) $R^{R \circ S} = \left(1,3\right), \left(3,5\right)$, $R^{S \circ R} = \left(1,3\right), \left(2,4\right), \left(2,5\right), \left(1,4\right), \left(1,5\right)$
3)S è la chiusura transitivà di se stessa

**Comment by niccolo-fato** [comment]
@LuginiAndrea una domanda me teoricamente alla domanda 1 dove chiede se R⋂S è transitiva, la risposta non dovrebbe essere che non è transitiva visto che ad esempio (1,3) e (3,5) appartengono ad R⋂S ma (1,5) non appartiene a R⋂S.

**Comment by LuginiAndrea** [comment]
> @LuginiAndrea una domanda me teoricamente alla domanda 1 dove chiede se R⋂S è transitiva, la risposta non dovrebbe essere che non è transitiva visto che ad esempio (1,3) e (3,5) appartengono ad R⋂S ma (1,5) non appartiene a R⋂S.

(3,5) non appartiene all'intersezione
